conf-template/starport.conf

```diff
@@ -45,6 +45,11 @@ krux.starport {
     # slack webhook to be used (optional)
     slack_webhook_url = ???
 
+    # metrics engine (optional)
+    # Valid values: null, "graphite", "cloudwatch"
+    # Note that only one option can be specified
+    metric.engine = null
```

Comment:
👍

Reply:
@realstraw That's a fair point. I was trying to ensure that only one config is ever used, even if the config file contain valid information for both Graphite and Cloudwatch. `metricEngine` is then fed to `metricConfig` so that the correct config is fetched.